onnxruntime/contrib_ops/cpu/bert/group_query_attention.cc

```diff
@@ -114,17 +115,29 @@
     rotary_params.seq_stride = head_size;
     rotary_params.head_stride = sequence_length * rotary_params.seq_stride;
     rotary_params.batch_stride = (packed_qkv ? (num_heads_ + 2 * kv_num_heads_) : num_heads_) * rotary_params.head_stride;
-    rotary_params.position_ids_format = sequence_length == 1 ? 1 : 0;
+    rotary_params.position_ids_format = parameters.is_interactive || !parameters.is_prompt ? 1 : 0;
```

Comment:
>is_interactive [](http://example.com/codeflow?start=51&length=14)

looks like we don't need parameters.is_interactive

Reply:
since it is used in cuda ep i think we should leave it in cpu ep for readability, especially since we may expand continuous decoding to batch_size > 1 in the future. it also makes group_query_attention_helper.h more readable right now.